control/server.py

```diff
@@ -147,6 +159,14 @@ def _add_server_listener(self):
             self.server.add_insecure_port("{}:{}".format(
                 gateway_addr, gateway_port))
 
+    def _exception_spdk(self):
+        timeout = self.config.getfloat("spdk", "timeout")
+        stdout, stderr = self.spdk_process.communicate(timeout=timeout)
+        return_code = self.spdk_process.returncode
+        self.logger.error(f"SPDK return code: {return_code}")
+        self.logger.error(f"SPDK stdout:\n {stdout}")
+        self.logger.error(f"SPDK stderr:\n {stderr}")
```

Comment:
```suggestion
        if stdout:
            self.logger.error(f"SPDK stdout:\n {stdout}")
        if stderr:
            self.logger.error(f"SPDK stderr:\n {stderr}")
```

Reply:
BTW, maybe it's just me but I hate multi-line logs, as they complicate log regex/parsing :(

What I did in the past was either:
```python
self.logger.error(f"SPDK stderr:\n {stderr.replace('\n', '\\n')}")
```
Or
```python
for line in stderr.split():
    self.logger.error(f"SPDK stderr: {line}")
```